ubuntu_20.04-x86_64-coverity/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
+
+ARG COVERITY_TOKEN
+
+ENV COVERITY_PROJECT=odp
+
+RUN apt-get update
+
+RUN apt-get install -yy --no-install-recommends \
+  software-properties-common \
+  dirmngr \
+  gnupg-agent
+
+RUN apt-get update --fix-missing
+
+RUN apt-get install -yy \
+  autoconf \
+  automake \
+  ccache \
+  clang \
+  curl \
+  gcc \
+  gcc-10 \
+  git \
+  libconfig-dev \
+  libssl-dev \
+  libtool \
+  libstdc++-10-dev \
+  make \
+  sudo
```

Comment:
The following packages should also be installed to enable building/running all examples and validation test:
`libcli-dev libcunit1-dev libpcap-dev`

Reply:
Ok, how do we enable building of those examples and tests? We don't need to run them, as that is not needed for Coverity and thus should be done in other jobs, but building would add to Coverity.

Reply:
The configure step in ODP build will detect these dependencies and if they are available, the particular examples and validation tests are built automatically.

Reply:
Ah, so we only need to add the packages and they will be built.

Reply:
Yes